src/test/java/org/prebid/server/bidder/adnuntius/AdnuntiusBidderTest.java

```diff
@@ -97,6 +99,42 @@ public void makeHttpRequestsShouldReturnErrorWhenSomeImpExtCouldNotBeParsed() {
                 .allMatch(errorMessage -> errorMessage.startsWith("Unmarshalling error:"));
     }
 
+    @Test
+    public void makeHttpRequestsShouldReturnRequestsWithMaxDealsIfMaxDealsIsBiggestThatZero() {
+        // given
+        final BidRequest bidRequest = givenBidRequest(
+                givenImp(ExtImpAdnuntius.builder().maxDeals(10).build(), identity()));
+
+        // when
+        final Result<List<HttpRequest<AdnuntiusRequest>>> result = target.makeHttpRequests(bidRequest);
+
+        // then
+        assertThat(result.getErrors()).hasSize(0);
```

Comment:
`.isEmpty()` is better than `.hasSize(0)` as its easier to read. Please, replace with call to `.isEmpty()` and also check for similar occurences.